app/entities/query.go

```diff
@@ -16,22 +16,32 @@ type QueryParams struct {
 
 // Validate checks if QueryParameter fields are valid.
 func (qp QueryParams) Validate() error {
+	if qp.Limit == "" {
+		return fmt.Errorf("limit argument has to be set")
+	}
+
 	if _, err := strconv.Atoi(qp.Limit); err != nil {
 		return fmt.Errorf("limit argument has to be a number")
 	}
 
+	if qp.Offset == "" {
+		return fmt.Errorf("offset argument has to be set")
+	}
+
 	if _, err := strconv.Atoi(qp.Offset); err != nil {
 		return fmt.Errorf("offset argument has to be a number")
 	}
 
-	allowedSortArgs := []string{"first_name", "last_name", "created_at", ","}
+	allowedSortArgs := []string{"first_name", "last_name", "created_at"}
+
+	const sep = ","
```

Comment:
September?

Reply:
[sep](https://www.abbreviations.com/abbreviation/separator) Nice try 😃

Reply:
btw it is not only my imagination. 
according to *stdlib*:
```go
func Join(elems []string, sep string) string
```

Reply:
maybe "separ" ? )))

Reply:
Wait, how'd you know? 😄

Reply:
Constant name can be longer than func argument name.

Reply:
Constant names always should be descriptive. Func arguments names may be less descriptive, cos func documentation can contains some additional information and make understanding to be easier.

Reply:
It is a huge point. thanks